DEX.md

```diff
@@ -0,0 +1,63 @@
+# What is a DEX?
+
+A Decentralized EXchange (DEX) is a smart contract that enables users to trade tokens without relying on a centralized intermediary. Instead, trades are executed directly on-chain through liquidity pools.
+
+# Liquidity Pool:
+
+Each DEX consists of one or more liquidity pools. A pool typically holds two tokens (say, $T_0$ and $T_1$). 
+In an ideal world, there would be a pool for every possible token pair.
```

Comment:
No strong opinion, but we could also omit this line I think (except if you had something particular in mind why this is important - in which case maybe we can make this clearer?)